src/components/About.tsx

```diff
@@ -0,0 +1,150 @@
+import React, {useState} from 'react';
+import {
+  Modal,
+  StyleSheet,
+  Text,
+  Pressable,
+  View,
+  TouchableOpacity,
+  Image,
+  Linking,
+} from 'react-native';
+import {EXT_URL, DB_URL} from '../constants/constants';
+
+const About = () => {
+  const [modalVisible, setModalVisible] = useState(false);
+  return (
+    <View style={styles.centeredView}>
+      <Modal
+        animationType="slide"
+        transparent={true}
+        visible={modalVisible}
+        onRequestClose={() => {
+          setModalVisible(!modalVisible);
+        }}>
+        <View style={styles.centeredView}>
+          <View style={styles.modalView}>
+            <Pressable
+              style={null}
+              onPress={() => setModalVisible(!modalVisible)}>
+              <Image
+                style={styles.imgStyle}
+                source={require('../img/close.png')}
+              />
+            </Pressable>
+
+            <Text>
+              Created By: {'\n'}
+              <Image source={require('../img/khalislogo150.png')} />
```

Comment:
For assets, can we use a single source of file, like we did in `constants.ts`

Reply:
resolved